.github/workflows/testing.yml

```diff
@@ -355,3 +355,28 @@ jobs:
       - name: Itemized validation
         if: ${{ !cancelled() }}
         run: bin/test_itemcheck
+
+  discoverynode:
+    name: Discovery Node
+    if: ${{ !cancelled() && vars.TARGET_PROJECT != '' }}
+    needs: [ baseline, sequencer, redirect, runlocal ]
+    env:
+      TARGET_PROJECT: ${{ vars.TARGET_PROJECT }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - name: install pytest
+        run: |
+          pip install pytest
```

Comment:
Can/should this just be folded into bin/setup_base?  Seems like it would be simpler overall...

Reply:
Good point, it was already in setup_base